pkg/gofr/datasource/mongo/mongo.go

```diff
@@ -79,14 +78,24 @@ func (c *Client) Connect() {
 	m, err := mongo.Connect(context.Background(), options.Client().ApplyURI(uri))
 	if err != nil {
 		c.logger.Errorf("error connecting to mongoDB, err:%v", err)
-
+		return
+	}
+	err = m.Ping(context.Background(), nil)
+	if err != nil {
+		c.logger.Errorf("error connecting mongoDB %s, err:%v", hidePass(uri), err)
 		return
 	}
 
 	mongoBuckets := []float64{.05, .075, .1, .125, .15, .2, .3, .5, .75, 1, 2, 3, 4, 5, 7.5, 10}
 	c.metrics.NewHistogram("app_mongo_stats", "Response time of MONGO queries in milliseconds.", mongoBuckets...)
 
 	c.Database = m.Database(c.config.Database)
+	c.logger.Logf("connecting to mongoDB at %v to database %v", hidePass(uri), c.config.Database)
```

Comment:
We can only mention the host and port while connecting no need to show other information